src/core/store.ts

```diff
@@ -48,10 +93,31 @@ type StateListener = () => void
 type MountedAtoms = Set<AnyAtom>
 
 // store methods
+/**
+ * Read an atom's hidden [AtomState]. Derived atom state may be recomputed if they
+ * are invalidated and any of their transitive dependencies have changed.
+ */
 export const READ_ATOM = 'r'
+/**
+ * Write an atom's Value.
+ */
 export const WRITE_ATOM = 'w'
+/**
+ * Commit pending writes to an atom.
+ */
 export const COMMIT_ATOM = 'c'
+/**
+ * Add a subscriber function to an atom. The atom will become "mounted" if this
+ * is the first subscriber.
+ *
+ * The subscriber is called whenever the atom is *invalidated* (i.e. when it's
```

Comment:
To be precise, a subscriber is called when an atom is invalidated or the atom value is directly changed by `atom.write`.